recipes/arg_router/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, rmdir
+from conan.tools.cmake import CMakeDeps, CMakeToolchain, CMake
```

Comment:
Python imports are not need when doing https://github.com/conan-io/conan-center-index/pull/16736/files#diff-1a1b075f66c13b970d742d4a961ea2dd786a694bf163890db08441044834e34aR22

```suggestion
from conan.tools.cmake import CMake
```